extra/man/alacritty-msg.scd

```diff
@@ -0,0 +1,59 @@
+ALACRITTY-MSG(1)
+
+# NAME
+
+alacritty-msg - Send messages to Alacritty
+
+# SYNOPSIS
+
+This command communicates with running Alacritty instances through a socket,
+making it possible to control Alacritty without directly accessing it.
+
+# OPTIONS
+
+*-s, --socket* <socket>
+	Path for IPC socket communication
+
+# MESSAGES
+
+*create-window*
+	Create a new window in the same Alacritty process
+
+		*OPTIONS*
+			*--hold*
+				Remain open after child process exits
+
+			*--working-directory* <working-directory>
+				Start the shell in the specified working directory
+
+			*command* <command>...
+				Command and args to execute (must be last argument)
```

Comment:
I feel like it looks better without extra `\t`.

Reply:
Which part?

Reply:
Like all I've selected, so `OPTION` is not shifted.

Reply:
You can look at how `config` and its `OPTION` looks, it's indented correctly.